arm/Microsoft.Storage/storageAccounts/.bicep/nested_container_rbac.bicep

```diff
@@ -1,12 +1,12 @@
-param roleAssignment object
+param roleAssignmentObj object
 param blobContainerName string
 param storageAccountName string
 param builtInRoleNames object
 
-resource container_rbac 'Microsoft.Storage/storageAccounts/blobServices/containers/providers/roleAssignments@2020-04-01-preview' = [for principalId in roleAssignment.principalIds: {
-  name: '${storageAccountName}/default/${blobContainerName}/Microsoft.Authorization/${(empty(roleAssignment) ? guid(storageAccountName) : guid(storageAccountName, blobContainerName, principalId, roleAssignment.roleDefinitionIdOrName))}'
+resource container_rbac 'Microsoft.Storage/storageAccounts/blobServices/containers/providers/roleAssignments@2020-04-01-preview' = [for principalId in roleAssignmentObj.principalIds: {
```

Comment:
resource roleAssignment

Reply:
Fixed, should be ok. Please resolve if you agree.

Reply:
I'd replace `container_rbac` with `roleAssignment` but let's take storage account outside of this PR